src/firebaseConfig.js

```diff
@@ -1,5 +1,6 @@
-import * as firebase from "firebase";
```

Comment:
u shouldn't delete line 1 and 2

Reply:
You need to add these back: `import * as firebase from "firebase";
import "firebase/database";`